.github/CODE_OF_CONDUCT.md

```diff
@@ -1,6 +1,8 @@
 # Code of Conduct
 
-This project follows the Code of Conduct defined in [this document][code_of_conduct]. Please read the full text for understanding the accepted and unaccepted behavior. Please read also the [reporting guidelines][guidelines], in case you encountered or witnessed any misbehavior.
+This project follows the Code of Conduct defined in [this document][code_of_conduct].
```

Comment:
I'd love to find something that would make "Code of Conduct" a link instead of "this document".